elfpy/utils/apeworx_integrations.py

```diff
@@ -139,7 +161,81 @@ def get_on_chain_trade_info(hyperdrive: ContractInstance) -> OnChainTradeInfo:
     for block_number_, price in share_price_.items():
         logging.debug(("block_number_={}, price={}", block_number_, price))
 
-    return OnChainTradeInfo(hyper_trades_, unique_maturities_, unique_ids_, unique_block_numbers_, share_price_)
+    return OnChainTradeInfo(trades, unique_maturities_, unique_ids_, unique_block_numbers_, share_price_)
+
+
+def log_subtotals(balance, query_balance, hyper_trades, idx, asset_type):
```

Comment:
I don't think you really need this function, since it's only used once.

Reply:
I only added it to get below the local variable limit per function. Let me see if I can get rid of it.

Reply:
I just got rid of it, since it was only used for debugging. To debug in the future, we can revive it from this PR. Or just re-do it, since it was just some fancy print statements.